.vsts-ci.yml

```diff
@@ -124,15 +126,18 @@ stages:
         agentOs: Windows_NT_TestAsTools
         pool:
           ${{ if eq(variables['System.TeamProject'], 'public') }}:
-            vmImage: 'windows-2019'
+            name: $(DncEngPublicBuildPool)
+            demands: ImageOverride -equals 1es-windows-2022-open
           ${{ if ne(variables['System.TeamProject'], 'public') }}:
             name: $(DncEngInternalBuildPool)
-            demands: ImageOverride -equals windows.vs2019.amd64
+            demands: ImageOverride -equals 1es-windows-2022
```

Comment:
Using the default AzDO pool was hitting out-of-space errors. Setting this to the 1ES pool resolved the issue.